recipe/meta.yaml

```diff
@@ -26,7 +24,7 @@ build:
   # Tensorflow still hasn't updated from C++14.
```

Comment:
This comment seems out of date now? Last status was that the newer abseil builds are all C++17, right?

Reply:
I think your understanding is correct. Lets get this merged in then we can move it forward.